README.md

```diff
@@ -42,7 +42,7 @@ async fn main() {
     let retry_policy = ExponentialBackoff::builder().build_with_max_retries(3);
     let client = ClientBuilder::new(reqwest::Client::new())
         // Trace HTTP requests. See the tracing crate to make use of these traces.
-        .with(TracingMiddleware)
+        .with(TracingMiddleware::default())
```

Comment:
I'd suggest to use the fully qualified syntax here to make it easier for users to then adapt the invocation when they need custom spans - i.e. `TracingMiddleware::< DefaultSpanBackend>::new()`

Reply:
There is already an example of a custom span in the `reqwest-tracing` README.
https://github.com/TrueLayer/reqwest-middleware/pull/43/files#diff-4ef5ea57cfc232174dcd3985f9f2aca2f3dd55cb90ae786019a8bfda2d7666faR68